scripts/start-server.sh

```diff
@@ -38,6 +38,7 @@ fi
 $KEYCLOAK/bin/add-user-keycloak.sh -u admin -p admin
 # Start the server
 $KEYCLOAK/bin/standalone.sh -Djava.net.preferIPv4Stack=true \
-                            -Djboss.socket.binding.port-offset=100 > keycloak.log 2>&1 &
+                            -Djboss.socket.binding.port-offset=100 \
+                            -Dkeycloak.profile.feature.upload_scripts=enabled > keycloak.log 2>&1 &
```

Comment:
Remove "-Dkeycloak.profile.feature.upload_scripts=enabled". QuickStarts should not rely on deprecated features, and Pedro has a PR open to change the quickstart.

Reply:
Yes, I believe @pedroigor 's Pull Request (https://github.com/keycloak/keycloak-quickstarts/pull/163) should be rebased on top of this one to make sure all failures (caused by the script policies) have been addressed. 

We might do it the other way around as well but since my PR is green, testing Pedro's changes is super easy (just rebase them on top of my commit and make sure it's still green).